### PR TITLE
EXT-727: Create NPM version of QuickStart ExtJS example

### PR DESCRIPTION
**Pull request by unnikrishnan-narayanan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,12 +3,9 @@
...
 
-## Download the Ext JS Framework
-
-If you have not already done so, download and unpack the Ext JS framework from
-either the Products section of the main Sencha website
-([www.sencha.com](http://www.sencha.com)) or from the downloads section of the
-Sencha Support portal.
+## Getting started
+### Prerequisite
+- Install [Node.js](https://nodejs.org/) (^6.9.2)
```
Comment (dalley) [inline README.md]: Do we have an actual dependency on 6.9 or can we update the docs to point to the latest?

File: app.json [hunk 2]
```diff
@@ -0,0 +1,144 @@
+{
+  "name": "QuickStart",
+  "namespace": "QuickStart",
+  "version": "1.0.0.0",
```
Comment (dalley) [inline app.json]: This version seems incorrect.

File: app.json [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    "appCache": {"enable": false},
+    "resources": {
+      "path": "${build.id}/resources",
+      "shared": "resources"
+    }
+  },
+  
+  "fashion": {
+		"missingParameters": "error",
+    "inliner": {"enable": false}
```
Comment (dalley) [inline app.json]: There are some minor lint issues in this file, spacing above is incorrect.

File: app/desktop/src/Application.scss [hunk 4]
```diff
@@ -0,0 +1,92 @@
+// $deep-blue: #32404e;
```
Comment (dalley) [inline app/desktop/src/Application.scss]: @unnikrishnan-narayanan  Do we need all these comments?  Or can they just be deleted?

File: app/desktop/src/view/main/MainViewController.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+		vm.set('detailCollapsed', !vm.get('detailCollapsed'));
+		if(vm.get('detailCollapsed')===true) {
+			button.setIconCls('x-fa fa-arrow-left');
+		}
+		else {
+			button.setIconCls('x-fa fa-arrow-right');
+		}
+	}
+
+//	onActionsViewLogoutTap: function( ) {
```
Comment (dalley) [inline app/desktop/src/view/main/MainViewController.js]: I can't tell what is new vs, what is being moved over from the old repo.  But in general, I think we can drop unused code.

File: README.md (hunk 1 above)
Comment (unnikrishnan-narayanan) [reply README.md]: it says upwards of 6.9.2 and I didn't change it from what it was.

File: app/desktop/src/Application.scss (hunk 4 above)
Comment (unnikrishnan-narayanan) [reply app/desktop/src/Application.scss]: These are coming in via the ext-gen app as example I think. for now, I'll leave as is

**Pull request merged**